CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 set(SOURCE_FILES ${SRC_PATH}/redis-roaring.c
         ${SRC_PATH}/type.c
-        ${SRC_PATH}/data-structure.c)
+        ${SRC_PATH}/data-structure.c
+        ${SRC_PATH}/sds.c)
```

Comment:
Maybe sds could be an external dependency as a git module, just like `CRoaring` and `hiredis` are?
https://github.com/antirez/sds